src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -304,7 +320,7 @@ private void PossiblyGlobArg(string arg, bool usedQuotes)
                     }
                 }
             }
-            else if (!usedQuotes)
+            else if (stringConstantType == StringConstantType.BareWord)
```

Comment:
This feels like it should be within the previous if clause:

```csharp
if (stringConstantType == StringConstantType.BareWord)
{
    if (WildcardPattern.ContainsWildcardCharacters(arg))
    {
        ...
    }
    else
    {
        ...
    }
}
```

Reply:
makes sense, will update